core/src/main/c/share/files.c

```diff
@@ -88,6 +88,18 @@ JNIEXPORT jlong JNICALL Java_io_questdb_std_Files_read
     return pread((int) fd, (void *) address, (size_t) len, (off_t) offset);
 }
 
+JNIEXPORT jlong JNICALL Java_io_questdb_std_Files_readULong
```

Comment:
there is existing function `TableUtils.readLongAtOffset()`

Reply:
I don't like hassle of allocation / de-allocating 8 bytes to read a long from a file which we do often, I thought this will be a good simplification